easybuild/tools/modules.py

```diff
@@ -1361,7 +1360,14 @@ def __init__(self, *args, **kwargs):
         setvar('LMOD_EXTENDED_DEFAULT', 'no', verbose=False)
 
         super(Lmod, self).__init__(*args, **kwargs)
-        self.supports_depends_on = StrictVersion(self.version) >= StrictVersion(self.REQ_VERSION_DEPENDS_ON)
+        version = StrictVersion(self.version)
+
+        self.supports_depends_on = version >= self.REQ_VERSION_DEPENDS_ON
+        # See https://lmod.readthedocs.io/en/latest/125_personal_spider_cache.html
+        if version >= '8.7.12':
```

Comment:
```suggestion
        if version >= StrictVersion('8.7.12'):
```
Not sure if this is necessary or not, but at least it keeps consistency with how other versions are being treated.

Reply:
No it is not necessary and in fact I changed the `self.supports_depends_on` to be consistent with this. I prefer this shorter variant when it is clear one operand is already a version instance (which is the case here as it is created right above)